packages/cna-template/template/nuxt/components/Tutorial.vue

```diff
@@ -0,0 +1,46 @@
+<!-- Please remove this file from your project -->
+<template>
+  <div class="relative flex items-top justify-center min-h-screen bg-gray-100 sm:items-center sm:pt-0">
+    <link href="https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css" rel="stylesheet">
```

Comment:
```suggestion
    <link href="https://cdn.jsdelivr.net/npm/tailwindcss@2.1.2/dist/tailwind.min.css" rel="stylesheet">
```

jsdelivr should be more stable for production. Also didn't checked but isn't `base.min.css` enough?

Reply:
I see two review regarding cdnjs or jsdelivr, what would be the good one?

I actually used the one recommended in TailwindCSS docs: https://tailwindcss.com/docs/installation#using-tailwind-via-cdn

Since it is for dev only I think it is fine.

Reply:
Both are good and use Cloudflare as a provider (jsdelivr backed by +5 more). I sometimes see downtimes with unpkg (especially since currently using `^2` range it is depending on resolver and we cannot leverage full cache. If you prefer unpkg, I would suggest at least using pinned version to avoid downtime chances and also better cache perf.

Reply:
/cc @Atinux